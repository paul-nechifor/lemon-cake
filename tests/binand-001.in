(& 12 6 13)
(& 0xff00000000000000 0x0f0000ff000000ff)
