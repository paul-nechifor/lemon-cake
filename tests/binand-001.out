4
1080863910568919040
